experimental/Schemes/AffineSchemes.jl

```diff
@@ -479,31 +481,31 @@ function issubset(
 end
 
 function issubset(
-    X::AbsSpec{BRT, <:MPolyQuoLocalizedRing}, 
+    X::AbsSpec{BRT, <:MPolyQuoLocalizedRing},
     Y::AbsSpec{BRT, <:MPolyLocalizedRing{<:Any, <:Any, <:Any, <:Any, <:MPolyPowersOfElement}}
   ) where {BRT}
   R = base_ring(OO(X))
   R == base_ring(OO(Y)) || error("schemes can not be compared")
   UX = inverted_set(OO(X))
   UY = inverted_set(OO(Y))
-  if !issubset(UY, UX) 
+  if !issubset(UY, UX)
     # check whether the inverted elements in Y are units anyway
     for a in denominators(UY)
       is_unit(OO(X)(a)) || return false
     end
   end
-  return iszero(OO(Y)(modulus(OO(X))))
+  return true
```

Comment:
I agree with the correctness of the change. For later reference, please comment in the code, why you do not need to worry about the modulus of X.